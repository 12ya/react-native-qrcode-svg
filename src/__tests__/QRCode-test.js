import React from 'react';
import QRCode from '../index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <QRCode />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
