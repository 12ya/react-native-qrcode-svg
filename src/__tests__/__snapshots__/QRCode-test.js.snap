exports[`test renders correctly 1`] = `
<div>
  <span>
    Rect
  </span>
  <span>
    Path
  </span>
</div>
`;
